.github/workflows/sync-upstream.yml

```
@@ -0,0 +1,163 @@
+  schedule:
+    # 每日 UTC 02:00 执行 (北京时间 10:00)
+    - cron: '0 2 * * *'
+  workflow_dispatch: # 允许手动触发
+
+jobs:
+  sync:
+    runs-on: ubuntu-latest
+    permissions:
+      contents: write
+      pull-requests: write
+    
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)